recon_surf/smooth_aparc.py

```diff
@@ -368,19 +370,19 @@ def smooth_aparc_files(
         incortexname: str,
         outaparcname: str
 ) -> None:
-    """Smoothes aparc.
+    """
+    Smoothes aparc.
```

Comment:
Read files, smooth the aparc labels on the surface and save the smoothed labels.